### ISSUE TITLE
Training failed

### ISSUE THREAD

**Issue by repustatemartin** [huggingface/autonlp#71]
Hello, 

Tried to train a model using AutoNLP and all 5 models failed. I verified the input CSVs are valid by loading them using datasets.

Any idea?

The project is called "appliances" and the owner is "mostrovsky"

**Comment by abhi1thakur** [comment]
Hi @repustatemartin ,

After a quick look at your project, there seems to be some inconsistencies in the label names and the number of labels in training and validation set. For example, one class is duplicated due to casing, you have 61 labels in training set but only 60 in validation set.

If you fix these problems and create a new project, the training should work perfectly fine.

Please let me know if I can help with something.

**Comment by repustatemartin** [comment]
Yes @abhi1thakur you're right, I'll fix that. Thanks for catching.

**Comment by SBrandeis** [comment]
I'm closing this since it seems resolved
